src/Kafka/AclsTreeView/AclsTreeView.tsx

```diff
@@ -0,0 +1,403 @@
+import React, { useRef, useState, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+
+import {
+  MenuToggle,
+  Menu,
+  MenuContent,
+  MenuGroup,
+  MenuList,
+  Popper,
+  TreeView,
+  TreeViewDataItem,
+} from "@patternfly/react-core";
+import { SolidLabel } from "../../shared/SolidLabel";
+import { AclResourceType, AclOperation } from "../../utils";
+
+export type AclsTreeViewProps = {
+  setSelectedItems: (items: TreeViewDataItem[]) => void;
+};
+
+export const AclsTreeView: React.VFC<AclsTreeViewProps> = ({
+  setSelectedItems,
+}) => {
+  const { t } = useTranslation("manage-kafka-permissions");
+
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [checkedItems, setCheckedItems] = useState<TreeViewDataItem[]>([]);
```

Comment:
I think we should be [lifting this state up](https://reactjs.org/docs/lifting-state-up.html). We will have a toolbar where this filter component will be displayed, and that toolbar will probably have a "Clear all filter" button to reset the filter to the initial state (which might be a previously saved one if that feature will get the ok).